apps/api/src/jobs/consumers/notifications/notification.consumer.ts

```diff
@@ -25,15 +25,51 @@ export abstract class NotificationConsumer {
     try {
       this.logger.log(`Sending notification with id: ${id}`);
       const result = await sendNotification();
-      notification.deliveryStatus = DeliveryStatus.SUCCESS;
+      notification.deliveryStatus = DeliveryStatus.AWAITING_CONFIRMATION;
```

Comment:
How to do plan to handle the case when we don't have to wait for confirmation for certain providers?

Reply:
This is now handled in https://github.com/OsmosysSoftware/osmo-x/pull/263/commits/664d10474f08085912604774e3ed18b9c6ed862b by adding a `SkipProviderConfirmationChannels` array in `constants/notifications.ts`